bundle/manifests/backstage-operator.clusterserviceversion.yaml

```diff
@@ -198,6 +201,11 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
+                env:
+                - name: POSTGRESQL_IMAGE
```

Comment:
we might need to prefix these variables with `RELATED_IMAGE_` so that the downstream build can automatically collect them into `spec.relatedImages`.

Reply:
Sounds good. Updated the  PR.